types/knockout/test/templatingBehaviors.ts

```diff
@@ -575,7 +575,7 @@ describe('Templating', function() {
 
         // Now set the observable to null and check it's treated like an empty array
         // (because how else should null be interpreted?)
-        myArray(null);
+        myArray(null as any); // DefinitelyTyped note: while KO accepts this, I wouldn't consider it a well-typed usage of the API
```

Comment:
Yup, I agree with this note. You might consider adding: `// $ExpectError` instead of `as any` above this so the test runner expects this to now be a type error.